### PR TITLE
Update Project Card Content and Styling

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This pull request makes visual changes to the project section.

- **Update the project cards to reflect project information**
- **Add colouring and styling to page**
- **Update hover effects**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -72,6 +72,7 @@ a {
 
 #projects {
   align-items: center;
+  background-color:var(--secondary);
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`, here and elsewhere

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -83,12 +84,15 @@ a {
   background: white;
   border-radius: 0.5rem;
   border: 1px black solid;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [use `0` instead of `0px`, if possible](https://google.github.io/styleguide/htmlcssguide.html#0_and_Units)

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Updated with commit 95c7166

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Updated with commit ea2db77

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -104,6 +108,7 @@ a {
 
 .card-subheading {
   filter: invert(1);
+  display:none;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -112,10 +117,12 @@ a {
 
 .card:hover>.card-subheading {
   filter: invert(0);
+  display:initial;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -112,10 +117,12 @@ a {
...
 }
 
 .card:hover>.card-heading {
   filter: invert(1);
+  display:none;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`

**Review by krasimirgg — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Rectified in commit here and in other places with 4c79bad

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Rectified in commit here and in other places with 4c79bad

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Rectified in commit here and in other places with 4c79bad

**Review by krasimirgg — APPROVED** [review]
Cool!

**Pull request merged**